src/main/java/core/basesyntax/strategy/Operations.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
+
+public enum Operations {
```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Operation {
```